### ISSUE TITLE
Moveing Tiles only work with Game Layer?

### ISSUE THREAD

**Issue by teini94** [ddnet/ddnet#6083]
I did add those arrows with Game Layer
5 Arrows one Under the "Unfreeze Laser"(with Font Layer)

but when I add those Arrows with Font Layer
and the "Unfreeze Laser" with Game layer it didnt work"
![screenshot_2022-12-03_22-55-07](https://user-images.githubusercontent.com/110184616/205463911-81f3c1a8-50c9-4ce3-bce0-1655645e0b1e.png)

**Comment by fokkonaut** [comment]
Yup. You need to place them from Game Layer. You can simply place the entity from Front Layer.

**Comment by Robyt3** [comment]
If those tiles don't work in the front layer then the editor shouldn't allow placing them there.
